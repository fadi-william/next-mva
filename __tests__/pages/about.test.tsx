import * as React from "react";
import * as renderer from "react-test-renderer";
import AboutPage from "../../pages/about";

// Mock the i18n local storage module.
jest.mock("../../app/storage/i18n");

it("about page - snapshot test", () => {
  const tree = renderer.create(<AboutPage />).toJSON();

  expect(tree).toMatchSnapshot();
});
