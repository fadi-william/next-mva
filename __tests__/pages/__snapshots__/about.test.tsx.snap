// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`about page - snapshot test 1`] = `
<div
  style={
    Object {
      "border": "1px solid #DDD",
      "margin": 20,
      "padding": 20,
    }
  }
>
  <div
    className="header"
  >
    <nav>
      <a
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
      <a
        href="/about"
        onClick={[Function]}
      >
        About
      </a>
      <div
        className="langSwitcher"
      >
        <select
          defaultValue="en"
          onChange={[Function]}
        >
          <option
            value="fr"
          >
            French
          </option>
          <option
            value="en"
          >
            English
          </option>
        </select>
      </div>
    </nav>
  </div>
  <p>
    This is the about page!
  </p>
</div>
`;
